frontend/app/src/pages/user/login/index.vue

```diff
@@ -92,6 +93,7 @@ onMounted(async () => fetchMessages());
         <p v-if="false" class="text-body-2 text-rui-primary">
           {{ t('login.welcome_update_message') }}
         </p>
+        <NewReleaseChangelog v-if="showReleaseNotes" class="mt-4" />
         <WelcomeMessageDisplay
```

Comment:
maybe the second one should be v-else-if so that they don't both show at the same time?